src/gui/qgsrelationeditorwidget.h

```diff
@@ -267,6 +267,9 @@ class GUI_EXPORT QgsRelationEditorWidget : public QgsAbstractRelationEditorWidge
     bool mAllowAddChildFeatureWithNoGeometry = true;
     QString mFilterExpression;
 
+    // Mainly used for testing purposes
+    QgsFeatureRequest mFeatureRequest;
```

Comment:
can we not get it from mDualView->masterModel()->request() instead of storing/copying it here?

Reply:
agreed and done